pyfar/classes/audio.py

```diff
@@ -203,10 +203,14 @@ def __init__(self, data, times, comment=None):
             must match the `size` of the last dimension of `data`.
         comment : str, optional
             A comment related to `data`. The default is ``'none'``.
+        complex : bool
+            A flag which indicates if the raw data are real or complex-valued.
+            The default is ``False``.
         """
 
         _Audio.__init__(self, 'time', comment)
 
+        self._complex = complex
```

Comment:
I'd suggest to add a small check, that _complex_ is actually a bool before setting the private variable.